src/playground-components/weather-element.html

```diff
@@ -0,0 +1,55 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/app-storage/app-localstorage/app-localstorage-document.html">
+<link rel="import" href="../../bower_components/iron-ajax/iron-ajax.html">
+<link rel="import" href="../shared-styles.html">
+<link rel="import" href="../redux-store.html">
+
+<dom-module id="weather-element">
+    <template>
+        <style>
+            #temp {
+                text-align: center;
+                font-style: italic;
+            }
+        </style>
+
+        <h3 style="text-align: center">Click the button below to fetch the current temperature in St. Louis, MO!</h3>
+        <div style="display:flex;justify-content:center;align-items:center;">
+            <paper-button id="weatherButton" raised on-click="__fetchWeather">Fetch Weather!</paper-button>
+        </div>
+        <template is="dom-if" if="{{temp}}">
```

Comment:
Braces are actually used for two-way data binding so for contexts in which data isn't being pulled out of a nested component the double square brackets should be used. There really is no harm when using curly braces but you don't want to risk two-way binding occurring. :-)

Reply:
Ahh got it thanks!!